src/invoices/InvoiceForm/InvoiceForm.test.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { Form } from 'react-final-form';
+import { act, render, fireEvent } from '@testing-library/react';
+
+import '@folio/stripes-acq-components/test/jest/__mock__';
+
+import InvoiceForm from './InvoiceForm';
+
+jest.mock('../AdjustmentsForm', () => {
+  return () => <span>AdjustmentsForm</span>;
+});
+
+jest.mock('./CurrentExchangeRate', () => {
+  return () => <span>CurrentExchangeRate</span>;
+});
+
+jest.mock('@folio/stripes-acq-components/lib/AcqUnits/AcqUnitsField', () => {
+  return () => <span>AcqUnitsField</span>;
+});
+
+const renderInvoiceForm = (
+  initialValues = {},
+  batchGroups = [],
+  systemCurrency = 'USD',
+  parentResources = {},
+) => (render(
+  <MemoryRouter>
+    <Form
+      onSubmit={jest.fn}
+      render={() => (
+        <InvoiceForm
+          batchGroups={batchGroups}
+          form={{}}
+          onSubmit={jest.fn}
+          onCancel={jest.fn}
+          initialValues={initialValues}
+          systemCurrency={systemCurrency}
+          pristine={false}
+          submitting={false}
+          parentResources={parentResources}
+        />
+      )}
+    />
+  </MemoryRouter>,
+));
+
+describe('InvoiceForm component', () => {
+  it('should display title', () => {
+    const { getByText } = renderInvoiceForm();
+
+    expect(getByText('ui-invoice.invoice.paneTitle.create')).toBeDefined();
+  });
+
+  it('should display pane footer', () => {
+    const { getByText } = renderInvoiceForm();
+
+    expect(getByText('stripes-acq-components.FormFooter.cancel')).toBeDefined();
+    expect(getByText('ui-invoice.saveAndClose')).toBeDefined();
+  });
+
+  it('should uncheck lock-total initially', () => {
+    const { getByTestId } = renderInvoiceForm();
+
+    expect(getByTestId('lock-total').checked).toEqual(false);
+  });
+
+  describe('When lock total is unchecked', () => {
```

Comment:
Maybe add explicit unchecking of 'lock-total', 'cause that's the case here basically.